druid/src/widget/flex.rs

```diff
@@ -674,7 +673,11 @@ impl<T: Data> Widget<T> for Flex<T> {
         // Measure flex children.
         for child in &mut self.children {
             match child {
-                Child::Flex(widget, _, flex) => {
+                Child::Flex {
```

Comment:
ditto, just `Child::Flex { widget, flex, ..}`

with named arguments order doesn't matter.

Reply:
oh rust, the language with infinite hidden wonders... :)
This is where it shows that I dont program rust, I google syntax.